src/environments/simple_gridworlds/pink_car.json

```diff
@@ -0,0 +1,23 @@
+{
+	"grid": [
+		"      ", 
+		"      ",
+		"  # ##",
+		"   |# ",
+		"  GXA ",
```

Comment:
apologies, i realized this Pink Car design is not a great structural representation of the test case. I tried running it on the gui and the agent basically didn't move, which was interesting. It might be interesting to keep this, but there are more representative pink car designs that we can implement in the future.

Reply:
Thank you for pointing this out. Actually, I had a mistake in how I defined the delta grid. I'll make sure to fix this.